src/rockstor/system/nut.py

```diff
@@ -146,7 +235,7 @@ def establish_config_defaults(config):
     """
     # An empty config dictionary will be false so raise exception as what else.
     if not config:
-        # todo I am unsure if this is the best way to raise an exception here.
+        # TODO: I am unsure if this is the best way to raise an exception here.
```

Comment:
This is fine, but a more pythonic approach would be to just assume that config is properly passed here and let the interpreter raise the KeyError if that is not the case. The caller of this method or higher up in the view, can raise this exception if the contract of properly configuring the service is broken.